pkg/oauth/user.go

```diff
@@ -192,16 +187,32 @@ func (s *server) Login(c echo.Context) error {
 		return err
 	}
 	userIDs := ttnpb.UserIdentifiers{UserID: req.UserID}
+	tokenID, err := auth.GenerateID(ctx)
+	if err != nil {
+		return err
+	}
+	tokenKey, err := auth.GenerateKey(ctx)
+	if err != nil {
+		return err
+	}
+	hashedKey, err := auth.Hash(ctx, tokenKey)
```

Comment:
What is the added benefit of also hashing this ? The bytes returned by `GenerateKey` are just random bytes (it's not a derived key by any means).

Reply:
We're doing the same thing for API keys and access tokens as well. I guess the rationale is that the actual keys do not get exposed in case of a DB leak.